ethereum/spec/src/test/java/tech/pegasys/teku/spec/datastructures/util/BeaconStateUtilTest.java

```diff
@@ -20,9 +20,9 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static tech.pegasys.teku.spec.datastructures.util.BeaconStateUtil.compute_next_epoch_boundary;
+import static tech.pegasys.teku.spec.datastructures.util.BeaconStateUtil.compute_shuffled_index;
```

Comment:
if this is only called by the test should it just be removed, or tests moved to the right spot to test the new function?

Reply:
It's used within `BeaconStateUtil`.  This test is for the deprecated `BeaconStateUtil`, so it will all get deleted when we get rid of `BeaconStateUtil`.